community/security/src/main/java/org/neo4j/server/security/auth/AuthProcedures.java

```diff
@@ -40,7 +40,7 @@
     public AuthSubject authSubject;
 
     @Description( "Create a user." )
-    @Procedure( name = "dbms.createUser", mode = DBMS )
+    @Procedure( name = "dbms.security.createUser", mode = DBMS )
     public void createUser( @Name( "username" ) String username, @Name( "password" ) String password,
             @Name( "requirePasswordChange" ) boolean requirePasswordChange )
```

Comment:
The community version seems to be missing the default value. Should be the same signature and default as enterprise.